pkg/issuer/acme/issue.go

```diff
@@ -398,9 +398,10 @@ func buildOrder(crt *v1alpha1.Certificate, csr []byte) (*v1alpha1.Order, error)
 		return nil, err
 	}
 
+	// truncate certificate name so final name will be <= 63 characters
 	return &v1alpha1.Order{
 		ObjectMeta: metav1.ObjectMeta{
-			Name:            fmt.Sprintf("%s-%d", crt.Name, hash),
+			Name:            fmt.Sprintf("%.52s-%d", crt.Name, hash),
```

Comment:
Where does 52 come from? Is the has always 11 characters long? Can you add a comment to justify the number somewhere, so we don't forget why it's the case in future? 😄

Reply:
I think I was going to but I got distracted :sweat_smile: